src/index.js

```diff
@@ -9,13 +9,14 @@ function err (node) {
     throw node.parent.error('No background color was found.', {plugin: plugin});
 }
 
-function getYIQContrast (colour) {
+function getYIQContrast (colour, colours) {
     let [r, g, b] = color(colour).rgbArray();
     let yiq = ((r * 299) + (g * 587) + (b * 114)) / 1000;
-    return (yiq >= 128) ? '#000' : '#fff';
+    return (yiq >= 128) ? colours.dark : colours.light;
 }
 
-export default postcss.plugin(plugin, () => {
+export default postcss.plugin(plugin, (opts) => {
+    if (typeof opts === 'undefined') opts = {dark: '#000000', light: '#ffffff'};
```

Comment:
I would prefer using [object-assign](https://github.com/sindresorhus/object-assign) for this:

```js
opts = assign({
    dark: '#000',
    light: '#fff'
}, opts);
```